File: back/src/main/java/com/pingo/entity/keywords/Keyword.java
package com.pingo.entity.keywords;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class Keyword {
    private String kwId;
    private String kwName;
    private String kwParentId;
    private String kwMessage;
    private String kwLevel;
}

File: front/lib/data/models/keyword_model/keyword.dart
class Keyword {
  String? kwId;
  String? kwName;
  String? kwParentId;
  String? kwMessage;
  String? kwLevel;

  Keyword(
      {required this.kwId,
      required this.kwName,
      required this.kwParentId,
      required this.kwMessage,
      required this.kwLevel});

  Map<String, dynamic> toJson() {
    return {
      'kwId': kwId,
      'kwName': kwName,
      'kwParentId': kwParentId,
      'kwMessage': kwMessage,
      'kwLevel': kwLevel,
    };
  }

  Keyword.fromJson(Map<String, dynamic> json)
      : kwId = json['kwId'],
        kwName = json['kwName'],
        kwParentId = json['kwParentId'],
        kwMessage = json['kwMessage'],
        kwLevel = json['kwLevel'];
}
